test/files/pos/existentials.scala

```diff
@@ -6,17 +8,44 @@ class A {
 
   val quux0 = f()
   def quux1 = f()
-  // lazy val quux2 = f()
-  // def quux3 = {
-  //   lazy val quux3a = f()
-  //   quux3a
-  // }
+  lazy val quux2 = f()
+  def quux3 = {
+    lazy val quux3a = f()
+    quux3a
+  }
 
+  // spurious warning until scala/bug#10612, cf test/files/neg/t7187.scala
   val bippy0 = f _
   def bippy1 = f _
-  // lazy val bippy2 = f _
-  // val bippy3 = {
-  //   lazy val bippy3a = f _
-  //   bippy3a
-  // }
+  lazy val bippy2 = f _
+  /*
+  val bippy3 = {
+    lazy val bippy3a = f _
+    bippy3a
+  }
```

Comment:
`bippy3` compiles just fine in 2.13.0-M3; better sneak the test case in before it regresses.

```scala
Welcome to Scala 2.13.0-M3 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_121).
Type in expressions for evaluation. Or try :help.

scala> def f() = { case class Bob(); Bob }
f: ()Bob.type forSome { val Bob: scala.runtime.AbstractFunction0[Bob] with Serializable{case def unapply(x$0: Bob): Boolean}; type Bob <: Product with Serializable{def copy(): Bob} }

scala>

scala> :pa
// Entering paste mode (ctrl-D to finish)

  val bippy3 = {
    lazy val bippy3a = f _
    bippy3a
  }


// Exiting paste mode, now interpreting.

bippy3: Bob.type forSome { val Bob: scala.runtime.AbstractFunction0[Bob] with Serializable{case def unapply(x$0: Bob): Boolean}; type Bob <: Product with Serializable{def copy(): Bob} } = Bob

scala>
```

Reply:
we can un-comment it in 2.13.x after this PR has been merged forward.

Reply:
Yep, that was what I meant. 👍